modules/dnn/src/layers/crop_and_resize_layer.cpp

```diff
@@ -130,16 +130,16 @@ class CropAndResizeLayerImpl CV_FINAL : public CropAndResizeLayer
         dims[3] = 7;
 
         auto lower_bounds = std::make_shared<ngraph::op::Constant>(ngraph::element::i64,
-                                             ngraph::Shape{offsets.size()}, offsets.data());
+                                             ngraph::Shape{offsets.size()}, std::vector<int64_t>(offsets.begin(), offsets.end()));
         auto upper_bounds = std::make_shared<ngraph::op::Constant>(ngraph::element::i64,
-                                             ngraph::Shape{dims.size()}, dims.data());
+                                             ngraph::Shape{dims.size()}, std::vector<int64_t>(dims.begin(), dims.end()));
```

Comment:
I believe we should change this line above instead:

```
std::vector<size_t> dims = rois->get_shape(), offsets(4, 0);
```

Reply:
I changed the fix.